features/hashchange.yml

```
@@ -0,0 +1,9 @@
+name: HashChangeEvent
+description: "The `HashChangeEvent` is emitted when the fragment identifier of the current page URL has changed."
```

Mostly for consistency with other event feature descriptions, but also to explain to folks who don't remember what fragment refers to:

```suggestion
description: "The `hashchange` event fires when the URL fragment identifier (the part of the URL starting with `#`) of the current page has changed."
```

---

Aghgh I went back and forth on this one